io/net/net_data.py

```diff
@@ -232,8 +232,8 @@ def tearDown(self):
         if 'yes' in self.eth_state:
             process.system("ifconfig %s up" % self.iface, shell=True)
             if self.ip_set:
-                process.system("ifdown %s" % self.iface, shell=True)
-                process.system("ifup %s" % self.iface, shell=True)
+                process.system("ifconfig %s down" % self.iface, shell=True)
```

Comment:
ifconfig is / will be deprecated in some distros. use `ip` command instead.
Ubuntu supports netplan, that can also be used.
Btw, ifup/ifdown can be installed in ubuntu.